clifford/test/test_function_cache.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+from numba import generated_jit
+import warnings
+
+# Make the test fail on a failed cache warning
+warnings.simplefilter("error")
```

Comment:
When put here, this will affect the whole test suite.

Reply:
I was a bit worried about whether that might be the case